src/main/java/wolox/training/models/User.java

```diff
@@ -0,0 +1,99 @@
+package wolox.training.models;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Date;
+import java.util.List;
+
+import javax.persistence.CascadeType;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.JoinTable;
+import javax.persistence.ManyToMany;
+import javax.persistence.Table;
+import javax.validation.constraints.NotNull;
+
+import wolox.training.exceptions.BookAlreadyOwnedException;
+
+@Entity
+@Table(name="Users")
+public class User{
+	
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private long id;
+    
+    @NotNull
+    private String userName;

```

Comment:
In unique values ​​like username can be added
@Column (unique = true) on the value in question to declare it in the database.

```suggestion
    @Column (unique = true)
    private String userName;

```

Reply:
La card no menciona que sea unique, solo requerida

Reply:
Tenes razón la card no lo dice que tiene que ser único. Te lo dejo entonces para que ya lo sepas.